### ISSUE TITLE
Cities in deal screen not showing correctly

### ISSUE THREAD

**Issue by an-accident** [Azurency/CQUI_Community-Edition#410]
In deal screen cities are usually incorectly named. Adding them to the trade shows the real names but it's not always possible: sometimes clicking just does nothing, even if the city hiding under the button should be tradeable (e.g. not capital).
Here an example:
![Zrzut ekranu (292)](https://user-images.githubusercontent.com/48416874/66929612-a2646e80-f033-11e9-9f28-11e37000047b.png)
Here cities are completly dis-named. Most of them named after my capital - Qusqu. As you can see, clicking one one of them puts in the central list the right city. and by 'the right' I mean the one that should alphabetically be in that position in the left list. But, for example, adding any of the following two Qusqu's to the trade is not possible.
Other symptoms:
-list does not depend on which leader I'm trying to deal with
-changing city name does not usually impact list (even next turn), loosing/aquiring does

**Comment by RajendranPalanisamy** [comment]
Yes, I am also facing the same issue. The opponent column reflect all of my cities' name/details instead of the opponent's cities name/details.

**Comment by the-m4a** [comment]
this should be fixed with the merge to the expansion2 branch that happened today (PR #415 included this fix)
